src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1426,5 +1426,13 @@ internal enum ErrorCode
         ERR_TypeInferenceFailedForImplicitlyTypedOutVariable = 8197,
         ERR_ExpressionTreeContainsOutVariable = 8198,
         #endregion diagnostics for out var
+
+        // gap here; available 8149-8499
+
+        #region diagnostics for instrumentation
+
+        ERR_InvalidInstrumentationKind = 8501,
```

Comment:
Do you expect to have bunch of new error codes for instrumentation?
If not, we just compacted the C#7 error codes, so we'd rather not introduce new large gaps. You could grab `8111`.